chain-signatures/node/src/indexer.rs

```diff
@@ -1,7 +1,5 @@
-use crate::gcp::error::DatastoreStorageError;
-use crate::gcp::GcpService;
 use crate::protocol::{SignQueue, SignRequest};
-use crate::types::LatestBlockHeight;
+use crate::storage::app_data_storage::AppDataRedisStorage;
```

Comment:
We should drop what the underlying db is for storage name since it's unneeded info:

```suggestion
use crate::storage::app_data_storage::AppDataStorage;
```